doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -154,3 +272,133 @@ Here is the offline configuration tool workflow:
    :align: center
 
    offline tool workflow
+
+.. _acrn_config_tool_ui:
+
+How to use ACRN config app
+**************************
+ACRN config app is used to read board info, configure and validate scenario settings,
+generate board source code and scenario source code, configure and validate launch
```

Comment:
automatically generate a patch for board related configuration, scenario-based VM configuration